File: 1. Backend/VehicleServiceStation/src/main/java/com/coforge/VehicleServiceStation/Models/Service.java
package com.coforge.VehicleServiceStation.Models;

import java.time.LocalDate;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.OneToOne;

import lombok.Data;

@Data
@Entity
public class Service {
	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	int id;

	@OneToOne
	JobCard jobcard;
	String type;
	String addParts;

	@OneToOne
	Employee employee;

	@OneToOne
	Feedback feedback;

	LocalDate date;

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	
}

File: 2. Frontend/VSS/src/app/_models/Service.ts
import { JobCard } from "./JobCard";
import { Feedback } from "./Feedback";
import { Employee } from "./Employee";


export class Service {
    id: number;
    jobcard: JobCard;
    type: string;
    addParts: string;
    employee: Employee;
    feedback: Feedback;
    date: Date;
}
